### PR TITLE
Added link to the text Apache 2.0 license in README

### PR DESCRIPTION
**Pull request by Satyxm**
### Description of your changes
I added a link to the text Apache 2.0 License which will redirect to the Apache 2.0 license.

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ for more information on how you can help.
 
 ## License
 
-Crossplane is under the Apache 2.0 license.
+Crossplane is under the [Apache 2.0 license](https://github.com/appvia/crossplane/blob/master/LICENSE).
```
Comment (jeanduplessis) [inline README.md:140]:
```suggestion
Crossplane is under the [Apache 2.0 license](https://github.com/crossplane/crossplane/blob/master/LICENSE).
```

**Review by jeanduplessis — CHANGES_REQUESTED** [review]
The link to the license file is currently pointing to the wrong GitHub org

File: README.md (hunk 1 above)
Comment (Satyxm) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ for more information on how you can help.
 
 ## License
 
-Crossplane is under the Apache 2.0 license.
+Crossplane is under the [Apache 2.0 license](https://github.com/crossplane/crossplane/blob/master/LICENSE).
```
Comment (jbw976) [inline README.md:140]: Would it be possible for you to follow the same "named links" pattern as most all other links in this file? See the named links section just a few lines below to show examples of how that pattern works :)

**Review by jbw976 — COMMENTED** [review]
Thank you for making this update to help with discoverability of the license file!

A couple meta comments:
* go ahead and rebase/squash so this PR is down to just 1 commit, keeping the history in main after this is merged nice and tidy
* there's some content in the upstream README that appears to be missing in your fork - looks like maybe you've forked from appvia's fork, which is more than [1K commits behind upstream](https://github.com/appvia/crossplane/compare/master...crossplane:crossplane:master).  I don't think this will cause a problem with the scoped nature of the changes here, but something to be aware of.

File: README.md (hunk 2 above)
Comment (Satyxm) [reply README.md:140]: If the current fork is 1K commits behind upstream then, should I fork the crossplane repo and create a new PR?

**Pull request closed without merging**